src/ast/node/function.rs

```diff
@@ -467,7 +549,7 @@ impl TypeNode for FuncDefNode {
                 fntype: FnType {
                     ret_pltype: self.ret.clone(),
                     param_pltypes,
-                    method,
+                    st_method: method,
```

Comment:
[Question]: what is the `st_method`? 

Could you consider to add a comment to doc this field?